bots/po-trigger

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+DAYS = 10
+
+import argparse
+import os
+import subprocess
+import sys
+
+sys.dont_write_bytecode = True
+
+import task
+
+def main():
+    parser = argparse.ArgumentParser(description="Ensure necessary issue exists for translations")
+    parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
+    parser.add_argument("context", nargs="?")
+    opts = parser.parse_args()
+
+    task.verbose = opts.verbose
+    text = "Update translations from Fedora Zanata"
+
+    if opts.context or task.stale(DAYS, "po/*.po"):
```

Comment:
Since stale doesn't check github for matching issues, do we need something to prevent multiple runners creating the same issue?